sonic_installer/common.py

```diff
@@ -20,10 +20,14 @@
 DOCKERDIR_NAME = 'docker'
 
 # Run bash command and print output to stdout
-def run_command(command):
-    click.echo(click.style("Command: ", fg='cyan') + click.style(command, fg='green'))
-
-    proc = subprocess.Popen(command, shell=True, text=True, stdout=subprocess.PIPE)
+def run_command(command, devnull=False):
+    command_str = join(command)
```

Comment:
should we reject a command when its type is string? Or, should we automatically shlex.split it if input command is a string?